### PR TITLE
[es] Translate Libraries > Poolboy section

### PR DESCRIPTION
**Pull request by j-burgos**
This pull request adds a translation to spanish for the Poolboy lesson in the Libraries section

### CODE REVIEW

File: es/lessons/libraries/poolboy.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+version: 1.1.2
+title: Poolboy
+---
+
+Puedes acabarte los recursos de tu sistema si no limitas el número de procesos concurrentes que tu programa puede crear.
+[Poolboy](https://github.com/devinus/poolboy) es una librería genérica de pooling para Erlang que es muy usada, ligera y que soluciona este problema.
+
+{% include toc.html %}
+
+## ¿Porqué utilizar Poolboy?
```
Comment (erickgnavar) [inline es/lessons/libraries/poolboy.md]: Porqué -> Por qué

File: es/lessons/libraries/poolboy.md [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+
+Pensemos por un momento en un ejemplo específico.
+Tienes la tarea de construir una aplicación que guarde la información del perfil de un usuario en una base de datos.
+Si creas un proceso por cada registro de usuario, estarías creando un número ilimitado de conexiones.
+En algún momento el número de conexiones podría llegar a sobrepasar la capacidad del servidor de base de datos.
+Eventualmente tu aplicación puede generar timeouts y varias excepciones.
+
+La solución es utilizar un conjunto de workers (procesos) para limitar el número de conexiones en lugar de crear un proceso por cada registro de usuario.
+Con eso fácilmente puedes evitar el acabarte los recursos de tu sistema.
```
Comment (erickgnavar) [inline es/lessons/libraries/poolboy.md]: `el` no sería necesario

File: es/lessons/libraries/poolboy.md [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+Todo lo que necesitamos hacer es agregar Poolboy como dependencia en nuestro archivo `mix.exs`.
+
+Creemos una aplicación primero
+
+```shell
+$ mix new poolboy_app --sup
+```
+
+Agregamos Poolboy como dependencia en nuestro archivo `mix.exs`.
+Add Poolboy as a dependency to our `mix.exs`.
```
Comment (erickgnavar) [inline es/lessons/libraries/poolboy.md]: Eliminar linea en inglés

**Comment by j-burgos** [comment]
Corrections have been done. Let me know of any other change :+1:

**Pull request merged**